docs/configuration/metrics/azure-queue.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Azure Queue Declaration
```

Comment:
Please rename `Azure Queue` to `Azure Storage Queue` as Azure also provides Service Bus Queues